catroid/src/main/java/org/catrobat/catroid/utils/UtilUi.java

```diff
@@ -86,4 +87,9 @@ public static void positionCursorForEditText(EditText groupNameEditText) {
 		Editable text = groupNameEditText.getText();
 		Selection.setSelection(text, position);
 	}
+
+	public static void showNoWebConnectionDialog(Context context) {
+        new NoWebConnectionDialogBuilder(context)
```

Comment:
im not a huge fan of one-line functions. either do new NoWebConnectionDialogBuilder(context).show() directly instead of calling this function (my preferred solution), or remove the NoWebConnectionDialogBuilder class and use the utility function to build the alert.